front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -1,5 +1,4 @@
-import React, { PureComponent } from 'react';
-import { adopt } from 'react-adopt';
+import React, { memo, useState } from 'react';
```

Comment:
Refactored to a functional component

Reply:
Yay! :)

Reply:
Sounds good. I don't know why we do this by default. In this component, I just did a refactor without thinking about whether the code was actually good. I'll remove the memo.